src/parser/jobs/blm/modules/Procs.tsx

```diff
@@ -58,7 +58,7 @@ export default class Procs extends CoreProcs {
 		const droppedThunderClouds: number = this.getDropCountForStatus(this.data.statuses.THUNDERCLOUD.id)
 		if (droppedThunderClouds > 0) {
 			this.suggestions.add(new Suggestion({
-				icon: this.data.actions.THUNDER_III_PROC.icon,
+				icon:  process.env.PUBLIC_URL + '/icon/action/t3p.png',
```

Comment:
since you use this twice, wouldn't it be better to define this as a const somewhere else?